modules/install-central-operator-tech-preview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * installing/install-ocp-operator.adoc
+:_module-type: PROCEDURE
+[id="install-central-operator-tech-preview_{context}"]
+= Installing Central with an external database using the Operator method
+
+:FeatureName: External PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+The main component of {product-title} is called Central. You can install Central on {ocp} by using the `Central` custom resource. You deploy Central only once, and you can monitor multiple separate clusters by using the same Central installation.
+
+[IMPORTANT]
+====
+When you install {product-title} for the first time, you must first install the `Central` custom resource because the `SecuredCluster` custom resource installation is dependent on certificates that Central generates.
+====
+
+.Prerequisites
+* You must be using {ocp} 4.6 or later.
```

Comment:
This just got changed recently to 4.10 or later so you might get a merge conflict. As of ACS 4.0, it's 4.10